fixnewlines.sh

```diff
@@ -0,0 +1 @@
+/home/jonny/fixnewlines.sh
```

Comment:
I find it useful to do a sweep through the complete diff before 
committing, since it helps catching extraneous files, trailing 
whitespace, duff/stale comments, and occasionally bugs.

On 2016-07-14 10:53, TMVector wrote:
> In fixnewlines.sh [1]:
> 
>> @@ -0,0 +1 @@
>> +/home/jonny/fixnewlines.sh
> 
> Yeah - I seem to have accidentally added a few files!
> 
> --
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub [2], or mute the
> thread [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/niksu/pax/pull/9#discussion_r70778868
> [2]
> https://github.com/niksu/pax/pull/9/files/7f29135a9261e8b25f7b9c3ec6cc9f4b3b0add3a..f7ac43cbd665a50aff815bdb5eb1eb81e6b9d99b#r70778868
> [3]
> https://github.com/notifications/unsubscribe/ABDTTuItDW63YUzt4lvk1ef6OPPuzAazks5qVgcOgaJpZM4JLXUX

-- 
http://www.cl.cam.ac.uk/~ns441/